pkg/products/rhssocommon/reconciler.go

```diff
@@ -127,7 +127,7 @@ func (r *Reconciler) CleanupKeycloakResources(ctx context.Context, inst *integre
 	if err != nil {
 		return integreatlyv1alpha1.PhaseFailed, err
 	} else if len(users.Items) > 0 {
-		logrus.Println("rhsso deletion of users in progress")
+		r.Log.Info("rhsso deletion of users in progress")
```

Comment:
Would it make sense to split it between user-sso and rhsso?

Reply:
The log will output with the context of user-sso or rhsso